addons/hr_expense/models/hr_expense.py

```diff
@@ -163,47 +162,33 @@ def _compute_state(self):
             else:
                 expense.state = "done"
 
-    @api.depends('quantity', 'unit_amount', 'tax_ids', 'currency_id')
+    @api.depends('quantity', 'unit_amount', 'tax_ids')
     def _compute_amount(self):
         for expense in self:
             if expense.product_id and not expense.unit_amount:
                 continue
-            taxes = expense._get_taxes(price=expense.unit_amount, quantity=expense.quantity)
+            taxes = expense._get_taxes(expense.unit_amount, expense.quantity, expense.currency_id)
```

Comment:
a value must be assigned everytime, remove the `continue`

Reply:
I am scared of changing this because then we also need to refactor _compute_from_product_id_company_id